desktop/src/main/java/bisq/desktop/main/overlays/windows/downloadupdate/DisplayUpdateDownloadWindow.java

```diff
@@ -260,6 +266,10 @@ private void addContent() {
                                             .onAction(() -> {
                                                 try {
                                                     Utilities.openFile(new File(Utilities.getDownloadOfHomeDir()));
+                                                    // We set a flag to clear tor cache files at re-start. We cannot clear it now as Tor is used and
+                                                    // that can cause problems.
+                                                    user.getCookie().putAsBoolean(CookieKey.CLEAN_TOR_DIR_AT_RESTART, true);
```

Comment:
I think we need to apply it also at close. So better move it directly into the `else` case so its set once download is completed.